homeassistant/components/modbus/__init__.py

```diff
@@ -239,6 +240,8 @@ def control_scan_interval(config: dict) -> dict:
     }
 )
 
+LIGHT_SCHEMA = SWITCH_SCHEMA
```

Comment:
The light integration doesn't support switch device classes.

Reply:
No, but it does not have its own, for a reason which I do not understand. Changed.